packages/shopify-codemod/transforms/arguments-to-args-spread.js

```diff
@@ -0,0 +1,65 @@
+export default function argumentsToArgsSpread({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+  function isArglessIIFE(path) {
```

Comment:
I find this name a little misleading because it's really looking for an IIFE that has no arguments **and** doesn't bind `arguments` (the arrow function part of it).